cluster/swarm/cluster.go

```diff
@@ -823,32 +823,32 @@ func (c *Cluster) TotalCpus() int64 {
 // Info returns some info about the cluster, like nb or containers / images
 func (c *Cluster) Info() [][2]string {
 	info := [][2]string{
-		{"\bStrategy", c.scheduler.Strategy()},
-		{"\bFilters", c.scheduler.Filters()},
-		{"\bNodes", fmt.Sprintf("%d", len(c.engines)+len(c.pendingEngines))},
+		{"Strategy", c.scheduler.Strategy()},
+		{"Filters", c.scheduler.Filters()},
+		{"Nodes", fmt.Sprintf("%d", len(c.engines)+len(c.pendingEngines))},
 	}
 
 	engines := c.listEngines()
 	sort.Sort(cluster.EngineSorter(engines))
 
 	for _, engine := range engines {
-		info = append(info, [2]string{engine.Name, engine.Addr})
-		info = append(info, [2]string{" └ Status", engine.Status()})
-		info = append(info, [2]string{" └ Containers", fmt.Sprintf("%d", len(engine.Containers()))})
-		info = append(info, [2]string{" └ Reserved CPUs", fmt.Sprintf("%d / %d", engine.UsedCpus(), engine.TotalCpus())})
-		info = append(info, [2]string{" └ Reserved Memory", fmt.Sprintf("%s / %s", units.BytesSize(float64(engine.UsedMemory())), units.BytesSize(float64(engine.TotalMemory())))})
+		info = append(info, [2]string{" " + engine.Name, engine.Addr})
+		info = append(info, [2]string{"  └ Status", engine.Status()})
```

Comment:
@riuvshin the point of this PR was to remove `\b` chars from the JSON. The proper offsets still need to be maintained for the Docker client to print this info appropriately. For now, you can be sure that just trimming the spaces is sufficient. Sorry about the confusion, if you're interested, #735 is the issue for a more structured JSON response which we'll tackle in the future.